recipes/whl2conda/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "whl2conda" %}
+{% set version = "23.8.4" %}
+
+package:
+  name: whl2conda
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/py3/w/whl2conda/whl2conda-{{ version }}-py3-none-any.whl
```

Comment:
The irony is not lost on me but can we build it from source instead of repacking the wheel?

Reply:
I am sure it is possible to build from source, but does it serve any purpose? I am not currently publishing sdists. There is already a wheel on pypi and so building from source is just busy work.

Really, I would like to just bypass conda build entirely and use this tool to build itself. I could do that if I were to publish to my own anaconda channel, but I don't know if that is even possible with the conda-forge infrastructure.

Reply:
There are other feedstocks that build from wheels, so I assumed it was kosher. E.g.

https://github.com/conda-forge/keras-feedstock/blob/main/recipe/meta.yaml

Reply:
Nope. It is a last resort measure and we really want to avoid it.

Reply:
Ok. Switched to build from sdist.